src/ModulesList.jsx

```diff
@@ -6,9 +6,9 @@ import fetch_modules from './fetch_modules';
 
 function getNodeVersion() {
   try {
-    return GETVal('node_version') == '4' ? '4' : '8';
+    return GETVal('node_version') == '8' ? '8' : '12';
```

Comment:
nitpick:
- Can we use a dictionary for mapping the versions numbers?
- Can we define constants for the default version that is returned upon exception?